cleanlab/classification.py

```diff
@@ -724,18 +722,14 @@ def find_label_issues(
         this method only requires a classifier and it can do the cross-validation for you.
         Both methods return the same boolean mask that identifies which examples have label issues.
         This is the preferred method to use if you plan to subsequently invoke:
-        :py:meth:`CleanLearning.fit()
-        <cleanlab.classification.CleanLearning.fit>`.
+        `~cleanlab.classification.CleanLearning.fit`.
```

Comment:
Explicitly specify `CleanLearning.fit()` here since the `fit` method exists in other classes as well (i.e. stick to :py:func: format).